pilot/pkg/bootstrap/server.go

```diff
@@ -360,6 +360,7 @@ func initOIDC(args *PilotArgs, trustDomain string) (security.Authenticator, erro
 	if err != nil {
 		return nil, fmt.Errorf("failed to unmarshal JWT rule: %v", err)
 	}
+	log.Infof("The JWT rule is %v", jwtRule)
```

Comment:
If you want to keep this - more context. 

"Istiod authenticating using JWTRule: ..."

Reply:
Revised according to the comment.